hw/ip/entropy_src/data/entropy_src.hjson

```diff
@@ -70,6 +77,67 @@
         }
       ]
     },
+    { name: "ES_CONF",
+      desc: "ENTROPY_SRC configuration register",
+      swaccess: "rw",
+      hwaccess: "hro",
+      regwen:   "ES_REGEN",
+      fields: [
+        { bits: "1:0",
+          name: "ENABLE",
+          desc: '''Master enable for the ENTROPY_SRC entropy generation function.
+                  This two bit field is the status of the application seed input FIFO.
+                  0b00: Disabled
+                  0b01: LFSR mode enabled
+                  0b10: Analog mode enabled
+                  0b11: Reserved
+                '''                
+        }
+        { bits: "4",
+          name: "ANA_SRC_EN",
+          desc: '''Setting this bit enables the analog entropy source. This should be set first,
+                wait for the status bit to set, then set the mode enable bits.
+                '''
+        }
+        { bits: "5",
+          name: "RCT_EN",
+          desc: "Setting this bit enables the health test called Repeative Count Test."
+        }
+        { bits: "6",
+          name: "APT_EN",
+          desc: "Setting this bit enables the health test called Adaptive Proportion Test."
+        }
+      ]
+    },
+    { name: "ES_RCT_HEALTH",
```

Comment:
are these cut-off values meant to be set and forget? If so, it seems like these should be under the effects of a `regwen` as well.

Reply:
Yes, set and forget - will add regwen.